controller/Dockerfile

```diff
@@ -1,5 +1,11 @@
 ARG BUILDPLATFORM=linux/amd64
 
+FROM --platform=$BUILDPLATFORM docker.io/curlimages/curl:7.75.0  as await
+WORKDIR /tmp
+ARG TARGETARCH=amd64
```

Comment:
`$TARGETARCH` is passed automatically, taking the values `amd64`, `arm64` and `arm` successively, so you might wanna remove this line.

Reply:
That wasn't happening when using bin/docker-build...

Reply:
For the multiarch build to be triggered you need to do
```
DOCKER_MULTIARCH=1 DOCKER_PUSH=1 bin/docker-build
```
`DOCKER_PUSH=1` is required because of a buildx bug (I haven't checked lately if that's still required), so be aware you'll be publishing the images.

Reply:
My point is that running a `bin/docker-build` as I normally do produced unusable images because the Linkerd-await program can't be downloaded. So I'd prefer to hardcode the default arg so this continues to work without requiring extra configuration (and pushing! ;).

Reply:
Ok I had forgotten this is just setting a default, not overriding things. LGTM :+1: